apps/pie-microsite/src/tokensMetadata.json

```
@@ -189,232 +189,383 @@
       "default": {
         "background-default": {
           "category": "pageBackgrounds",
-          "description": "Default page background"
+          "description": "Default page background",
+          "globalToken": "$color-mozzarella-5"
         },
         "background-subtle": {
-          "category": "pageBackgrounds"
+          "category": "pageBackgrounds",
+          "description": "Subtle page background",
+          "globalToken": "$color-mozzarella-10"
         },
         "background-dark": {
-          "category": "pageBackgrounds"
+          "category": "pageBackgrounds",
+          "description": "High contrast page background",
+          "globalToken": "$color-mozzarella-100"
         },
         "container-default": {
-          "category": "containerBackgrounds"
+          "category": "containerBackgrounds",
+          "description": "Default container background",
+          "globalToken": "$color-white"
         },
         "container-subtle": {
-          "category": "containerBackgrounds"
+          "category": "containerBackgrounds",
+          "description": "Subtle container background",
+          "globalToken": "$color-mozzarella-10"
         },
         "container-strong": {
-          "category": "containerBackgrounds"
+          "category": "containerBackgrounds",
+          "description": "Strong container background",
+          "globalToken": "$color-mozzarella-20"
         },
         "container-dark": {
-          "category": "containerBackgrounds"
+          "category": "containerBackgrounds",
+          "description": "Dark container background",
+          "globalToken": "$color-mozzarella-90"
         },
         "container-inverse": {
-          "category": "containerBackgrounds"
+          "category": "containerBackgrounds",
+          "description": "Inverse container background",
+          "globalToken": "$color-mozzarella-90"
         },
         "border-default": {
-          "category": "borders"
+          "category": "borders",
+          "description": "Default border",
+          "globalToken": "$color-mozzarella-20"
         },
         "border-subtle": {
-          "category": "borders"
+          "category": "borders",
+          "description": "Subtle border",
+          "globalToken": "$color-mozzarella-10"
         },
         "border-strong": {
-          "category": "borders"
+          "category": "borders",
+          "description": "Higher contrast border",
+          "globalToken": "$color-mozzarella-30"
         },
         "border-inverse": {
-          "category": "borders"
+          "category": "borders",
+          "description": "Border on inverse or dark coloured backgrounds",
+          "globalToken": "$color-mozzarella-70"
         },
         "divider-default": {
-          "category": "dividers"
+          "category": "dividers",
+          "description": "Divider for use above default and light coloured backgrounds",
+          "globalToken": "$color-black @8%"
```

I think this looks a little nicer? Happy to leave this for a small discussion later
```suggestion
          "globalToken": "$color-black @ 8%"
```

---

I'm not sure if I agree haha, sorry! Happy for us to discuss with Jordi at a later date though